include/common/aglShaderProgramArchive.h

```diff
@@ -22,11 +27,22 @@ class ShaderProgramArchive : public sead::IDisposer, public sead::hostio::Node {
     void updateCompileInfo();
     void setUp();
     void setUp_(bool);
+    void setUpFromObjectReflector(bool, bool);
+    int searchShaderProgramIndex(const sead::SafeString&) const;
+    sead::FormatFixedSafeString<1024> genMessage(sead::hostio::Context* context);
+    void listenPropertyEvent(const sead::hostio::PropertyEvent* property_event);
 
 private:
-    void* _20;
-    ResBinaryShaderArchive* mBinaryShaderArchive;
-    ResShaderArchive* mResShaderArchive;
+    ResBinaryShaderArchive mBinaryShaderArchive;
+    ResShaderArchive mResShaderArchive;
+    sead::Buffer<ShaderProgram> mShaderPrograms;
+    void* _48[6];
+    short _78[2];
+    sead::Buffer<ShaderProgramEdit> mShaderProgramEdits;
+    sead::Buffer<ShaderSource> mShaderSources;
+    sead::Buffer<void*> unkData;
+    sead::Buffer<bool*> unkData2;
```

Comment:
seems @aboood40091 has named these in his header
https://github.com/aboood40091/sead/blob/master/packages/agl/include/common/aglShaderProgramArchive.h#L147-L157

Reply:
While I did get some of these names from assertions and therefore they are confirmed to be correct, it's important to mention that I wasn't able to get ALL names and therefore some of these names were made up by myself. I have long forgotten which names are the confirmed ones and which are the made up ones.